lib/Serialization/ModuleFile.cpp

```diff
@@ -1666,6 +1675,9 @@ void ModuleFile::getImportDecls(SmallVectorImpl<Decl *> &Results) {
       if (Dep.isExported())
         ID->getAttrs().add(
             new (Ctx) ExportedAttr(/*IsImplicit=*/false));
+      if (Dep.isUsableFromInline())
+        ID->getAttrs().add(
+            new (Ctx) UsableFromInlineAttr(/*IsImplicit=*/false));
```

Comment:
This should be UsableFromInlineImportAttr…and should probably be marked implicit because we don't want to show them to users at this time.

Reply:
Sure. How can I test this?

Reply:
`test/IDE/print_clang_framework_with_overlay.swift` uses swift-ide-test to print a Clang framework's imports; you should be able to do the same sort of thing for a Swift module.

Reply:
Done.